.github/scripts/generate_ci_workflows.py

```diff
@@ -166,6 +166,7 @@ class CIWorkflow:
     num_test_shards_on_pull_request: int = -1
     distributed_test: bool = True
     fx2trt_test: bool = False
+    deploy_test: bool = False
```

Comment:
I think we don't need this; it should be controlled by the workflow name (e.g. in test.sh).

Reply:
same comment